tests/test_packages/test_connections/test_p2p_libp2p/test_aea_cli.py

```diff
@@ -37,27 +37,42 @@
     LIBP2P_CERT_NOT_BEFORE,
 )
 
-from tests.conftest import DEFAULT_LEDGER, DEFAULT_LEDGER_LIBP2P_NODE
+from tests.conftest import DEFAULT_LEDGER, DEFAULT_LEDGER_LIBP2P_NODE, LOCAL_HOST
 from tests.conftest import default_ports as ports
-from tests.conftest import libp2p_log_on_failure
+from tests.conftest import libp2p_log_on_failure_all
 
 
 p2p_libp2p_path = f"vendor.{p2p_libp2p.__name__.split('.', 1)[-1]}"
 DEFAULT_NET_SIZE = 4
 LIBP2P_LAUNCH_TIMEOUT = 20  # may downloads up to ~66Mb
 
 
-class TestP2PLibp2pConnectionAEARunningDefaultConfigNode(AEATestCaseEmpty):
-    """Test AEA with p2p_libp2p connection is correctly run"""
+class BaseP2PLibp2pConnectionAEATest(AEATestCaseEmpty):
+    """Base class for AEA CLI tests"""
+
+    capture_log = True
 
     @classmethod
     def setup_class(cls):
         """Set the test up"""
-        super(TestP2PLibp2pConnectionAEARunningDefaultConfigNode, cls).setup_class()
+        super().setup_class()
         cls.conn_key_file = os.path.join(os.path.abspath(os.getcwd()), "./conn_key.txt")
         cls.log_files = []
 
-    @libp2p_log_on_failure
+    @classmethod
+    def teardown_class(cls):
+        """Tear down the test"""
+        cls.terminate_agents()
+        cls.log_files.clear()
+        super().teardown_class()
```

Comment:
the AEA CLI tests do not use the base class that was introduced; they already have their own